test/TestServerProjectsLowLevel/body-complex/Generated/ArrayClient.cs

```diff
@@ -35,14 +35,14 @@ protected ArrayClient()
         /// <param name="endpoint"> server parameter. </param>
         /// <param name="options"> The options for configuring the client. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="credential"/> is null. </exception>
-        public ArrayClient(AzureKeyCredential credential, Uri endpoint = null, AutoRestComplexTestServiceClientsOptions options = null)
```

Comment:
Remind me how it decides which client to name the client options for?

Reply:
it uses the name of the library: https://github.com/Azure/autorest.csharp/pull/1842/files#diff-7d0053c8a00c13755fb7fe14b79cb3794ac143885f2e48e3b126cfb60d9308c6R59

Reply:
👍